frinx-uniconfig/user-guide/uniconfig-operations/kafka-notifications/readme.md

```diff
@@ -46,6 +47,41 @@ sent until either the NETCONF session or the subscription is terminated.
 NETCONF notifications are categorised as so-called streams. The subscriber
 must choose which streams to receive. The default stream is named *NETCONF*.
 
+## gNMI notifications
+
+[gNMI spec](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#35-subscribing-to-telemetry-updates) defines a
+mechanism where the gNMI client indicates an interest in receiving state of data instances via Subscribe RPC
+The gNMI server, if the creation of the subscription was successful, will reply with state of data instances according to specified paths. 
+The session will be opened until it will be closed either from UniConfig side (releasing of the subscription) 
```

Comment:
it would be good to mention that uniconfig tries to automatically recreate session

Reply:
ack